Documentation/api.md

```diff
@@ -0,0 +1,37 @@
+#### Api endpoint for V2
+
+There are two endpoints for the V2 versions of the endpoint:
+-`/api/v2/reports/{name}/full`
+-`/api/v2/reports/{name}/table`
+
+These are located in pkg/chargeback/http.go.
+
+{name} is the name if the report that you are looking to run. Formatting is specified as a query string at the end.
+
+#### Sample urls
+
+Replace namespaceVar with your namespace
+Replace reportnameVar with the name of your report
+Replace formatVar with json, csv or tabular
+
+#Full endpoint
+```
+$ curl "http://127.0.0.1:8001/api/v1/namespaces/namespaceVar/services/metering:http/proxy/api/v2/reports/reportnameVar/full?format=formatVar"
+```
+#Table endpoint
+```
+$ curl  "http://127.0.0.1:8001/api/v1/namespaces/namespaceVar/services/metering:http/proxy/api/v2/reports/reportnameVar/table?format=formatVar"
+
+```
+
+The endpoint `/api/v2/reports/{name}/full` performs a similar function to the `/api/v1/reports/get` endpoint, except the difference is in the formatting of the output display.
```

Comment:
This is weirdly positioned after table, but discusses full.

Reply:
You should provide an example URL that gets the following output. Eg: `/api/v2/reports/namespace-cpu-request/full?format=json`. 

Also explicitly describe the different in the formatting. Eg:

> The  `/api/v2/reports/{name}/full` endpoint returns reports in either CSV, JSON, or tabular format, similar to `/api/v1/reports/get`. The difference is in the structure of the JSON results. The JSON results from this endpoint contain more metadata about each field including the unit, and whether or not the field should be shown the in a table (used in the table endpoint below).
> 
> Sample output for a report as JSON (/api/v2/reports/namespace-cpu-request/full?format=json):
> {"results": ..... }

Then you should do the same for table indicating what it does.